### ISSUE TITLE
Quick command icons not visible anymore

### ISSUE THREAD

**Issue by Corbyn** [sni/Thruk#864]
The icons for the "Command:" entries in the quick command select box (_status_cmd_pane.tt) aren't visible anymore.
IE and Chrome never supported background images in an <option> of a <select> list, but current Firefox version also doesn't support it anymore.

Perhaps jquery could be used instead of html/css background-image:
http://www.marghoobsuleman.com/jquery-image-dropdown

**Thruk Version**
2.22

![image](https://user-images.githubusercontent.com/8446952/46817083-a1449580-cd7e-11e8-9ed3-ce38db4e578e.png)

**Comment by sni** [comment]
sounds like a bit overkill for a few images. Well, if no browser supports this anymore, i'd rather remove that code as well and just stick to text dropdown.

**Comment by Corbyn** [comment]
Yes, it is indeed. It's just that I tried all kind of html/css hacks for a couple of hours to get these icons back but it's apparently really not possible. So in the end I thought, I post a little issue to let you know. ;) But of course feel free to just close it...

**Comment by sni** [comment]
much appreachiated, i will close the issue as soon as i removed the code.
